src/nvim/globals.h

```diff
@@ -987,6 +987,8 @@ EXTERN char_u e_dirnotf[] INIT(= N_(
     "E919: Directory not found in '%s': \"%s\""));
 EXTERN char_u e_au_recursive[] INIT(= N_(
     "E952: Autocommand caused recursive behavior"));
+EXTERN char_u e_autocmd_close[] INIT(= N_(
+    "E813: Cannot close autocmd or popup window"));
```

Comment:
only `autocmd window` I think?

Reply:
Do we not use this for `floating` windows or do we not make a distinction between floating and regular windows for autocmds?

I don't mind removing `or popup` but I won't update the translations.

Reply:
AFAIK we only use this for autocmd window. Floats can always be closed just like regular windows.